packages/core/src/types/ViewportInputOptions.ts

```diff
@@ -10,6 +11,9 @@ type ViewportInputOptions = {
   orientation?: Orientation;
   /** whether the events should be suppressed and not fired*/
   suppressEvents?: boolean;
+  initialRange?: Point2;
+  canvasCenter?: Point2;
+  initialCenter?: Point2;
```

Comment:
All require proper documentation, we use these commments in the api doc https://www.cornerstonejs.org/api/core/namespace/Types#ViewportInputOptions